internal/venice-common/src/main/resources/avro/GlobalDivState/v-15/GlobalDivState.avsc

```diff
@@ -0,0 +1,85 @@
+{
```

Comment:
let's start with v1 and follow the folder naming convention

Reply:
Done.